app/views/items/show.html.erb

```diff
@@ -19,4 +19,8 @@
 
 <%= link_to "Add Review", new_item_review_path(@item) %>
 
+<% if @item.user == current_user %>
+  <%= link_to "Update Item", edit_item_path(@item) %>
+<% end %>
```

Comment:
You would also want to check that the item belongs to the user in the ItemsController#update method because `curl` exists.